widget/ui/src/components/Tooltip/Tooltip.styles.ts

```diff
@@ -17,3 +17,9 @@ export const TooltipTypography = styled(Typography, {
   boxShadow: '5px 5px 10px 0px rgba(0, 0, 0, 0.10)',
   backgroundColor: '$neutral100',
 });
+
```

Comment:
Can you check tooltip alignments?

![Screenshot from 2023-12-11 14-01-29](https://github.com/rango-exchange/rango-client/assets/99180934/d6d082e4-2833-466c-9190-f41361bb0adc)